src/shared/lib/pathwayMapper/PathwayMapperTable.tsx

```diff
@@ -31,7 +31,7 @@ interface IPathwayMapperTableProps {
     selectedPathway: string;
     changePathway: (pathway: string) => void;
     initialSortColumn?: string;
-    view: string;
+    columnsOverride?: any | null;
```

Comment:
Unless it is absolutely necessary we shouldn't be using `any` as a type. Here we can do something like this:

```suggestion
    columnsOverride?: {[columnEnum: number]: Partial<PathwayMapperTableColumn>};
```